docs/docs/guides/deploy/loadbalancing-example/example-traefik.yaml

```diff
@@ -59,5 +58,5 @@ http:
       loadBalancer:
         servers:
           # h2c is the scheme for unencrypted HTTP/2
-        - url: h2c://zitadel:8080
+          - url: h2c://zitadel:8080
```

Comment:
I do not think we should lint the yaml. Since we setup only md/mdx right now

Reply:
The formatter checks all .md files and everything in the docs folder atm

Reply:
Do you think formatting yaml is generally a bad idea?

Reply:
No not in general. But I am not sure of the scope of the lint.

Should we only lint md/mdx? Only things in /docs? Or everything (which would raise some questions on the console and go side)

Reply:
The idea of the scope I chose was to lint docs whereas I consider .md files outside the docs folder docs too. Maybe we find something more clear.

Reply:
Ok but in that case lets stick to md/mdx only.

Reply:
It hurts me a bit to exclude the yaml files, but I see the point 😄👍

Reply:
Well I see your point, but if we want to do this as well we should discuss our lint strategy in general.

Do we lint each component differently or do we apply rules across the full repo?!